src/DsaLocal.cc

```diff
@@ -974,6 +980,9 @@ void LocalAnalysis::runOnFunction(Function &F, Graph &g) {
         n.addAllocSite(a);
       // -- mark node as a stack node
       n.setAlloca();
+      // -- mark node as incomplete (formal arguments)
+      n.setIncomplete();
```

Comment:
this should be `markIncomplete` to spread the marking. The call should be done after `LocalAnalysis` terminates to ensure that everything reachable by an argument at the END is marked incomplete.